src/DynamoCore/Engine/EngineController.cs

```diff
@@ -38,6 +38,11 @@ public class EngineController : LogSourceBase, IAstNodeContainer, IDisposable
         /// </summary>
         public event AstBuiltEventHandler AstBuilt;
 
+        /// <summary>
+        /// This flag to to check if any new packages are being loaded and disable any execution run's that are triggered before the package loading is completed. 
```

Comment:
`This flag is used to check if any packages are currently being loaded, and to disable any executions that are triggered before the package loading is completed. See DYN-2101 for more info`